week-4/Homework/mandatory/4-groceries.js

```diff
@@ -60,3 +75,12 @@ let numberOfItemsPerWeak = {
   saturday: 0,
   sunday: 0
 };
+
+for( let daysOfWeek in numberOfItemsPerWeak){
+  for(day in weeklyMealPlan){
```

Comment:
This is a nice approach - you might also be able to solve this problem without using nested `for` loops though - this is because `numberOfItemsPerWeak` and `weeklyMealPlan` both have the same keys - and you can use them to access both objects in the same loop